hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/api/TokenServiceApiImpl.java

```diff
@@ -178,7 +176,9 @@ public void finalizeHollowAccountAsContract(@NonNull final AccountID hollowAccou
         }
         final var accountAsContract = hollowAccount
                 .copyBuilder()
-                .key(STANDIN_CONTRACT_KEY)
+                .key(Key.newBuilder()
+                        .contractID(ContractID.newBuilder().contractNum(hollowAccountId.accountNumOrElse(0L)))
```

Comment:
It seems like this should be,
```suggestion
                        .contractID(ContractID.newBuilder().contractNum(hollowAccountId.accountNumOrThrow()))
```
because we only use this [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/scope/HandleHederaNativeOperations.java#L138) and it explicitly converts to a `0.0.X` id,
```
final var hollowAccountId = requireNonNull(accountStore.getAccountIDByAlias(evmAddress));
```